tests/Unit/Service/Security/SensioLabsSecurityCheckerTest.php

```diff
@@ -0,0 +1,134 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Buddy\Repman\Tests\Unit\Service\Security;
+
+use Buddy\Repman\Service\Security\SecurityChecker\SensioLabsSecurityChecker;
+use PHPUnit\Framework\TestCase;
+use Symfony\Component\Filesystem\Filesystem;
+use Symfony\Component\Process\Exception\ProcessFailedException;
+
+final class SensioLabsSecurityCheckerTest extends TestCase
+{
+    private SensioLabsSecurityChecker $checker;
+    private string $dbDir;
+    private string $fixturesDir;
+
+    protected function setUp(): void
+    {
+        $this->dbDir = sys_get_temp_dir().'/repman/security-advisories';
+        $this->fixturesDir = __DIR__.'/../../../Resources/fixtures/security/locks';
+
+        $filesystem = new Filesystem();
+        $filesystem->mirror(
```

Comment:
do you have to make a copy of it? Can't point the path to these fixture directly?
```php
new SensioLabsSecurityChecker(__DIR__.'/../../../Resources/fixtures/security/security-advisories', 'bogus');
```

Reply:
during tests `.git` directory is created which may be left over in case of a test failure, interruption etc, this ensures that is always the same in the beginning